src/run.ts

```diff
@@ -103,11 +106,126 @@ async function run_cdk_watch(runner: LabeledProcessRunner, options: { stage: str
   await runner.run_command_and_output("CDK watch", watchCmd, ".");
 }
 
-async function run_local(options: { stage: string }) {
+function isDockerRunning() {
+  try {
+    execSync("docker info", { stdio: "ignore" });
+    return true;
+  } catch {
+    return false;
+  }
+}
+
+function isLocalStackRunning() {
+  try {
+    const output = execSync("localstack status", {
+      encoding: "utf-8",
+      stdio: "pipe",
+    }).trim();
+    return output.includes("running");
+  } catch {
+    return false;
+  }
+}
+
+async function run_watch(options: { stage: string }) {
   const runner = new LabeledProcessRunner();
   await prepare_services(runner);
+
   run_cdk_watch(runner, options);
-  run_fe_locally(runner, options);
+  run_fe_locally(runner);
+}
+
+async function run_local() {
+  const runner = new LabeledProcessRunner();
+  await prepare_services(runner);
+
+  if (!isDockerRunning()) {
+    throw "Docker needs to be running.";
+  }
+
+  if (!isLocalStackRunning()) {
+    throw "LocalStack needs to be running.";
+  }
+
+  process.env.AWS_DEFAULT_REGION = "us-east-1";
+  process.env.AWS_ACCESS_KEY_ID = "localstack";
+  process.env.AWS_SECRET_ACCESS_KEY = "localstack";
+  process.env.AWS_ENDPOINT_URL = "http://localhost:4566";
+
+  const cdklocalBootstrapCmd = [
+    "cdklocal",
+    "bootstrap",
+    "aws://000000000000/us-east-1",
+    "--context",
+    "stage=jon-cdk",
+  ];
+  await runner.run_command_and_output(
+    "CDK local bootstrap",
+    cdklocalBootstrapCmd,
+    "."
+  );
+
+  const deployLocalPrequisitesCmd = [
+    "cdklocal",
+    "deploy",
+    "--app",
+    '"npx tsx deployment/local/prerequisites.ts"',
+  ];
+  await runner.run_command_and_output(
+    "CDK local prerequisite deploy",
+    deployLocalPrequisitesCmd,
+    "."
+  );
+
+  const deployPrequisitesCmd = [
+    "cdklocal",
+    "deploy",
+    "--app",
+    '"npx tsx deployment/prerequisites.ts"',
+  ];
+  await runner.run_command_and_output(
+    "CDK prerequisite deploy",
+    deployPrequisitesCmd,
+    "."
+  );
+
+  const deployCmd = [
+    "cdklocal",
+    "deploy",
+    "--context",
+    "stage=jon-cdk",
+    "--all",
+    "--no-rollback",
+  ];
+  await runner.run_command_and_output("CDK deploy", deployCmd, ".");
+
+  const cloudFormationClient = new CloudFormationClient({
+    region: "us-east-1",
+  });
+  const command = new DescribeStacksCommand({ StackName: "seds-jon-cdk" });
+  const response = await cloudFormationClient.send(command);
+  const seedDataFunctionName = response.Stacks?.[0]?.Outputs?.find(
+    (output) => output.OutputKey === "SeedDataFunctionName"
+  )?.OutputValue;
+
+  const lambdaClient = new LambdaClient({ region: "us-east-1" });
+  const lambdaCommand = new InvokeCommand({
+    FunctionName: seedDataFunctionName,
+    InvocationType: "Event",
+    Payload: Buffer.from(JSON.stringify({})),
+  });
+  await lambdaClient.send(lambdaCommand);
```

Comment:
I wonder if we could put this in our cdk code like we did for bootstrapUsers. We could add the only-for-localstack-if around it if that is needed.

Reply:
:) this is in our CDK code, I had to convert it back right here only for localstack because we don't have localstack pro :)